content/en/docs/v3.5/op-guide/kubernetes.md

```diff
@@ -0,0 +1,445 @@
+      - name: etcd
+        image: quay.io/coreos/etcd:v3.5.15
+        imagePullPolicy: IfNotPresent
+        ports:
+        - name: etcd-client
+          containerPort: 2379
+        - name: etcd-server
+          containerPort: 2380
+        - name: etcd-metrics
+          containerPort: 8080
+        ##
+        ## These probes will fail over TLS for self-signed certificates, so etcd
+        ## is configured to deliver metrics over port 8080 further down.
+        ##
+        ## As mentioned in the "Monitoring etcd" page, /readyz and /livez were
+        ## added in v3.5.12. Prior to this, monitoring required extra tooling
+        ## inside the container to make these probes work.
+        ##
+        ## The values in this readiness probe should be further validated, it
+        ## is only an example configuration.
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: 8080
+          initialDelaySeconds: 10
+          periodSeconds: 5
+          timeoutSeconds: 5
+          successThreshold: 1
+          failureThreshold: 30
+        ## The values in this liveness probe should be further validated, it
+        ## is only an example configuration.
+        livenessProbe:
+          httpGet:
+            path: /livez
+            port: 8080
+          initialDelaySeconds: 15
+          periodSeconds: 10
+          timeoutSeconds: 5
+          failureThreshold: 3
+        env:
+        ##
+        ## Environment variables defined here can be used by other parts of the
+        ## container configuration. They are interpreted by Kubernetes, instead
+        ## of in the container environment.
+        ##
+        ## These env vars pass along information about the pod.
+        - name: K8S_NAMESPACE
+          valueFrom:
+            fieldRef:
+             fieldPath: metadata.namespace
+        - name: HOSTNAME
+          valueFrom:
+            fieldRef:
+             fieldPath: metadata.name
+        - name: SERVICE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.annotations['serviceName']
+        ##
+        ## Configuring etcdctl inside the container to connect to the etcd node
+        ## in the container reduces confusion when debugging.
+        - name: ETCDCTL_ENDPOINTS
+          value: $(HOSTNAME).$(SERVICE_NAME):2379
+        ##
+        ## TLS client configuration for etcdctl in the container.
+        ## These files paths are part of the "etcd-client-certs" volume mount.
+        # - name: ETCDCTL_KEY
+        #   value: /etc/etcd/certs/client/tls.key
+        # - name: ETCDCTL_CERT
+        #   value: /etc/etcd/certs/client/tls.crt
+        # - name: ETCDCTL_CACERT
+        #   value: /etc/etcd/certs/client/ca.crt
+        ##
+        ## Use this URI_SCHEME value for non-TLS clusters.
+        - name: URI_SCHEME
+          value: "http"
+        ## TLS: Use this URI_SCHEME for TLS clusters.
+        # - name: URI_SCHEME
+        # value: "https"
```

Comment:
Suggestion to add nested spaces for the value, so when removing the comment, the indentation is properly aligned (and matching the ones from lines 226-232).

```suggestion
        #   value: "https"
```

Reply:
The way it's written is so people can toggle the comments with hotkeys in most editors.